### ISSUE TITLE
AWK support incomplete

### ISSUE THREAD

**Issue by CameronD73** [hyperupcall/vscode-assorted-languages#8]
The awk language file includes a "source.regexp" grammar in 5 places, but this is not found on a default installation.
The result is that any regex, either in the main pattern or in the body of the code is not recognised at all and so is subject to incorrect pairing of quotes and probably brackets.
I don't know how to handle a missing dependency, and I could not find a suitable grammar file to install.  The nearest one was "[Injected Regex Syntax Highlighting](https://github.com/RedCMD/regex-syntax-highlighter-vscode) by redcmd. Using that language requires including a scope named `source.syntax.regexp`

**Comment by hyperupcall** [comment]
Thank you for your report - I'll look into it

**Comment by hyperupcall** [comment]
This has been fixed by a96507d419e5e730dcef72a0a8bc7ea764ea386a:

![image](https://github.com/fox-projects/vscode-assorted-languages/assets/24364012/b2c95e8f-e137-4d96-a835-321ff50a24c5)

**Comment by hyperupcall** [comment]
Thank you for noting https://github.com/RedCMD/regex-syntax-highlighter-vscode - I integrated that regex tmLanguage into this repository and have renamed it accordinly. I made its scope "under" awk because I suspect things will need to be changed to be adopeted for AWK. The syntax highlighting for awk regex can be improved, but for now I'll close this since any "bad syntax highlighting" is localized to the regex and no longer bleeds out to other areas of the syntax.
